_posts/2023-02-20-example-mapping-in-practice.markdown

```diff
@@ -0,0 +1,179 @@
+
+If only there was some way of that short conversation happening between developer, tester and product owner beforehand :wink:
+
+## The Neverending Story
+
+The opposite of the "She'll be right".
+
+Arming the developer and tester with all the information they could possibly want. This often occurs when someone writes a func spec in Given, When, Then syntax and places it into the Jira card to be agile.
+
+Given, When, Then, And Then......And Then And Then And Then
+
+- TLDR - Developer gives up and develops the feature that they want
+- Tester tests word for word rather than understanding what it's really supposed to do
+- Everyone gets into an argument
+
+In this scenario someone believed they knew the story very well, so documented it in great detail. Unfortunately they forgot to involve the rest of the team.
+
+## So why Example Mapping?
+
+What makes Example Mapping different is the process and the output:
+
+- You get the right people in the room...product owner, SMEs, developers and testers
+- You document the understanding of the user story in simple clear terms
+- With the above in place, you build a shared understanding
+- EVERYONE leaves the session knowing they could build & test the feature right now
+- The people that build the software build the requirements.....everyone wears the BA hat
+
+<br />
+
+# Examples of Examples
+
+Theres no set rules of how you should structure your examples, it really depends on the user story and rule, the following outlines some examples that I have seen and used myself. It doesn't always have to be the better known Given, When, Then syntax....whilst it works very well in a lot of scenarios its not the only way.
+
+## The Humble Table
+
+Tables are really underated, and they are great for showing examples of certain types of data. I have used tables to represent examples with many sessions I have run for example mapping.
+
+The following table elaborates on our Rule from above 'The discount rate is based off the previous month's after discount cost'
+
+![Example Table](/img/blog/examplemapping/humbletable.png)
+
+One huge advantage of tables is that they are easy to maintain in Excel/Sheets and you can add extra examples quickly during your virtual sessions - _"Yeh but Andrew....what about if Homer......"_
+
+## Fun with Fiction
+
+Adding names to roles can bring a hole load more fun to your example mapping sessions. A product owner once handed the team a matrix showing all the role -> function permissions for the system we were developing. This information was gold, and the team were super impressed with the effort put in, but in an attempt to create a quick reference example to facilitate conversations, we took the matrix and ran through it to create a simple pictorial view we could all talk to!
+
+Here is what our 'Kwik-E-Mart' app membership options might look like as a pictorial view.
+![Personas](/img/blog/examplemapping/funwithfiction.png)
```

Comment:
I like your point here, but you've lost the connection to the title (Fun with fiction).

I know, because I've got the context - but Lisa, Bart and Homer are already part of the Kwik-E-Mart 'domain'.